src/examples/wasm/server.go

```diff
@@ -3,15 +3,26 @@ package main
 import (
 	"log"
 	"net/http"
+	"os"
+	"path/filepath"
 	"strings"
 )
 
-const dir = "./html"
+var dir = "main"
 
 func main() {
-	fs := http.FileServer(http.Dir(dir))
+	if len(os.Args[1:]) == 1 {
```

Comment:
Simply `if len(os.Args) == 2` is easier to understand.

Reply:
Yeah.